src/haddock/modules/__init__.py

```diff
@@ -198,7 +198,8 @@ def _load_previous_io(self):
 
     def previous_path(self):
         """Give the path from the previous calculation."""
-        previous = sorted(list(self.path.resolve().parent.glob('[0-9][0-9]*/')))
+        # https://regex101.com/r/4qRTjn/1
+        previous = sorted(list(self.path.resolve().parent.glob('[0-9]*/')))
```

Comment:
Why does this need to be changed? It already parsed one or more digits which is what is needed. In this new version it parses zero or more digits which I think is not what is needed here. Maybe `\d+` is more logical, which is equivalent to `[0-9]+` which is equivalent to `[0-9][0-9]*`.